capabilityquerier/cmd/main.go

```diff
@@ -77,11 +77,12 @@ func queryEndpoints(message []byte, args *map[string]interface{}) error {
 	jobArgs := make(map[string]interface{})
 
 	jobArgs["querierArgs"] = capabilityquerier.QuerierArgs{
-		FhirURL:      fhirURL,
-		Client:       qa.client,
-		MessageQueue: qa.mq,
-		ChannelID:    qa.ch,
-		QueueName:    qa.qName,
+		FhirURL:       fhirURL,
+		FhirURLString: urlString,
```

Comment:
I don't think that we need _both_ FhirURL and FhirURLString to be on this structure. I think we can get away with only having the string version of the FhirURL, and then convert it to a url.URL when it needs to be of that type (right before the query to that URL occurs)

Reply:
I chnaged the Querier Args struct so that it only contains the string version of the FhirURL